merkletree/str.go

```diff
@@ -42,8 +43,8 @@ func NewSTR(key sign.PrivateKey, policies Policies, m *MerkleTree, epoch uint64,
 	return str
 }
 
-// Serialize encodes the STR to a byte array with the following format:
-// [epoch, previous epoch, tree hash, previous STR hash, policies serialization]
+// Serialize serializes the signed tree root into
+// the correct format for signing.
```

Comment:
I think it'd be preferable if we could say s/the correct/a specified/ but guess that's alright for now.